packages/vue/src/components/authenticator.vue

```diff
@@ -300,8 +298,11 @@ onBeforeMount(() => {
 
 const attrs = useAttrs();
 
-const { loginMechanisms } = withDefaults(
-  defineProps<{ loginMechanisms?: LoginMechanism[] }>(),
+const { initialState, loginMechanisms } = withDefaults(
+  defineProps<{
+    initialState: AuthenticatorMachineOptions['initialState'];
+    loginMechanisms: AuthenticatorMachineOptions['loginMechanisms'];
```

Comment:
These should be optional

```suggestion
    initialState?: AuthenticatorMachineOptions['initialState'];
    loginMechanisms?: AuthenticatorMachineOptions['loginMechanisms'];
```